src/components/SecondaryPanes/Frames/tests/Group.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { shallow } from "enzyme";
+import GroupComponent from "../Group.js";
+const Group = React.createFactory(GroupComponent);
+
+import FrameMenu from "../FrameMenu";
+jest.mock("../FrameMenu", () => jest.fn());
+
+function render(overrides = {}) {
+  const defaultProps = {
+    group: [{ displayName: "foo" }],
+    selectedFrame: {},
+    selectFrame: jest.fn(),
+    copyStackTrace: jest.fn()
+  };
+
+  const props = Object.assign({}, defaultProps, overrides);
+  const component = shallow(new Group(props));
+  return { component, props };
+}
+
+describe("Group", () => {
+  it("displays a group", () => {
+    const { component } = render();
+    expect(component).toMatchSnapshot();
+  });
+
+  describe("mouse events", () => {
+    it("calls FrameMenu on right click", () => {
+      const { component, props } = render();
+      const mockEvent = "mockEvent";
+      component.simulate("contextmenu", mockEvent);
+
+      expect(FrameMenu).toHaveBeenCalledWith(
```

Comment:
I don't feel strongly here, but I feel like testing the args is adding too much coupling. My concerns would be 

1. does it get called
2. does it return some menu items.

testing the communication seems like we could be introducing brittle tests...